app/views/layouts/hyrax/dashboard.html.erb

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="<%= I18n.locale.to_s %>" prefix="og:http://ogp.me/ns#">
+  <head>
+    <%= render partial: 'layouts/head_tag_content' %>
+    <%= content_for(:head) %>
+  </head>
+
+  <body data-spy="scroll" data-target=".bs-docs-sidebar" class="dashboard" style="position:relative">
```

Comment:
We should only need either this (the date-spy, data-target, and optionally data-offset attributes) or the JS.
If you go with JS, you'll have to set this body to `position: relative` some other way (which should be done anyways because inline-styles)